en/docs/deploy/cloud/deploy-from-cloud-editor.md

```
@@ -0,0 +1,49 @@
+
+   If you don't have a repository ready, click **Create New Repository** to create one.
+
+   If WSO2 Cloud does not have access to the repository, click **Connect Newly Created Repository** and grant access. This is required only if you have not already authorized WSO2 Cloud to access all repositories in your organization.
+
+   ![Deploy Integration tab showing repository configuration](/img/deploy/cloud/deploy-from-cloud-editor/adding-source-control.png)
+
+4. Click **Deploy**.
+
+WSO2 Cloud pushes the integration to the repository, builds it, and deploys it to the **Development** environment.
+
+{/* TODO: Add what's next after creating manage pages */}
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)